erpnext/stock/doctype/stock_settings/stock_settings.js

```diff
@@ -13,5 +13,11 @@ frappe.ui.form.on('Stock Settings', {
 
 		frm.set_query("default_warehouse", filters);
 		frm.set_query("sample_retention_warehouse", filters);
+	},
+	
+	before_save: function(frm){
+		if (!frm.doc.valuation_method){
+			frm.set_value("valuation_method", "FIFO");
+		}
```

Comment:
```suggestion
	before_save: function(frm){
		if (!frm.doc.valuation_method){
			frm.set_value("valuation_method", "FIFO");
		}
```
Set FIFO as default in stock settings json